source/guide_pocketbase.rst

```diff
@@ -0,0 +1,162 @@
+.. author:: bliepp <https://github.com/bliepp>
+
+.. tag:: lang-go
+.. tag:: audience-developers
+.. tag:: file-storage
+.. tag:: database
+.. tag:: authentication
+
+.. highlight:: console
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/pocketbase.svg
+      :scale: 300%
+      :align: center
+
+.. spelling:wordlist:
+  pocketbase
+  Pocketbase
+
+##########
+Pocketbase
+##########
+
+.. tag_list::
+
+Pocketbase_ is a self-hosted Backend as a Service with a functionality similar to Google Firebase or Supabase.
+It provides APIs for a managed SQLite database, user authentication, file storage and more.
+As most applications written in Go it's easy to install.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Web Backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+
+License
+=======
+
+Pocketbase is licensed under the MIT license
+
+     https://github.com/pocketbase/pocketbase/blob/master/LICENSE.md
+
+
+
+Installation
+============
+
+Download the latest release and extract it to your binary folder:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ wget -O pocketbase.zip https://github.com/pocketbase/pocketbase/releases/download/v0.22.8/pocketbase_0.22.8_linux_amd64.zip
+    [...]
+    Saving to: ‘pocketbase.zip’
+
+    100%[====================================================================================================================================================>] 14,321,393  23.1MB/s   in 0.6s
+
+    2024-04-11 11:22:32 (23.1 MB/s) - ‘pocketbase.zip’ saved [14321393/14321393]
+    [isabell@stardust ~]$ unzip pocketbase.zip pocketbase -d /home/isabell/bin/
+    Archive:  pocketbase.zip
+      inflating: /home/isabell/bin/pocketbase
+    [isabell@stardust ~]$
+
+It should be added to your path automatically. Now test if it works:
+
+.. code-block:: console
+
+    [isabell@stardust ~]$ pocketbase -v
+    pocketbase version 0.22.8
+    [isabell@stardust ~]$
+
+
+Setup Supervisord
+=================
+
+First create a folder where all data is stored:
+
+.. code-block:: console
+
+    [isabell@stardus ~]$ mkdir /home/isabell/pocketbase
```

Comment:
`mkdir ~/pocketbase` does the trick, no need for "isabell" here.

Reply:
Should I replace `/home/isabell` everywhere or only where you wrote the review comments?